packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,6 +2,72 @@
 
 ## Changelog
 
+### Apr 18, 2023
+
+New features:
+
+- Run and debug Teams bot and message extension applications with Dev Tunnel.
+  ![devtunnel](https://user-images.githubusercontent.com/11220663/230364699-ed108641-4196-4318-949b-17ff56a6593f.png)
+
+- `Validate Application` command can now catch publish errors and provide mitigation suggestions for the application package.
+  ![validate](https://user-images.githubusercontent.com/11220663/230363745-50bb8b6b-06a1-40df-b6ae-5601a71b193e.png)
+
+- Auto complete when editing the yml files.
+
+  > Note: Teams Toolkit for Visual Studio Code takes a new dependency on [YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) which will be installed together with Teams Toolkit.
+
+New getting started experience with Github Codespaces:
+
+Start developing Teams applications instantly by opening Teams Toolkit samples in Github Codespaces. Try these today：
+
+- [Hello world tab app]()
+- [Notification bot]()
+- [NPM search connector]()
+
+SharePoint Framework developer experience update:
+
+- Streamlined full stack debugging for SPFx based tab apps on Teams, Outlook and the Microsoft 365 app.
+- Automated the process to add multiple SPFx web parts as Teams Tab pages and configure the new web parts with Teams app.
+- Provided flexibility to let user decide which SharePoint Framework (SPFx) package version to use when creating a SPFx-based Teams tab app, either to choose from a globally installed version or let Teams Toolkit install the latest version for you.
+  ![SPFx](https://user-images.githubusercontent.com/11220663/230363218-98e16312-17b0-49cf-8617-333ca961c4b0.png)
+
+Teams Toolkit generated project file change:
+
+- Updated lifecycles in yml files to only include provision, deploy, and publish.
+- Some actions now support a `writeToEnvironmentFile` parameter to define generated environment variable names.
+- `.env.{envName}` files now only contain non-sensitive information and are removed from `.gitignore`.
+- Added `.env.{envName}.user` files to store sensitive information like password and are added to `.gitignore`.
+  ![project](https://user-images.githubusercontent.com/11220663/230363542-c0e8db66-8b6f-4c0e-92d0-b184c34e4824.png)
+
+Teams Toolkit CLI ([`@microsoft/teamsfx-cli`](https://www.npmjs.com/package/@microsoft/teamsfx-cli)) `v2.0.0@beta`:
```

Comment:
@nliu-ms @KennethBWSong @wenytang-ms is there any change to the non-lifecycle commands in CLI?

Reply:
For update aad-app command, there is no changes on interface and parameters, so no need to add to changelog